test/cucumber/steps/ParkingSpaceReminderSteps.groovy

```diff
@@ -0,0 +1,31 @@
+import static cucumber.api.groovy.EN.*
+
+// Cenario 1	
+Given(~'user "([^"]*)" parked on spot "([^"]*)" using the system$'){ String user, int numSpot ->

```

Comment:
Sintaxe incorreta, faltou o símbolo `^` no início do cenário para que o teste seja devidamente reconhecido.